.vuepress/code-samples/render.js

```diff
@@ -0,0 +1,39 @@
+const vuepressmd = require('@vuepress/markdown')()
+
+function codeBlockWrapper(sample, language) {
+  return `\`\`\` ${language}
+   ${sample}
+   \`\`\`
+  `
+}
+
+function renderCodeSample(sample, language) {
+  const wrappedSample = codeBlockWrapper(sample, language)
+  const htmlRender = vuepressmd.render(wrappedSample)
+  return htmlRender.html
+}
+
+module.exports = function(fetchedSamples) {
+  /**  This reduce creates an object containing all samples-id with a list of the samples in the different languages
+    ex: codeSamples = {
+      create_index_1 : [
+        { languages: "bash", code: "`$ curl -X GET 'http://localhost:7700/indexes", label: "curl" },
+        { languages: "javascript", code: "client.createIndex('movie')", label: "javascript" }
+      ]
+    }
+  **/
+  return Object.keys(fetchedSamples).reduce((acc, language) => {
+    for (const sampleId in fetchedSamples[language]) {
+      const previousSamples = acc[sampleId] ? acc[sampleId] : []
```

Comment:
Can't we write
```suggestion
      const previousSamples = acc[sampleId] || []
```
?

Reply:
I saw that practice a lot in ruby and JS 🙂

Reply:
Oh yes you're right! Good suggestion :)